src/error.rs

```diff
@@ -207,6 +217,9 @@ impl Display for Error {
             Error::DatabaseNotFound => {
                 write!(f, "Use of resolvers required a database back-end. Please select either cosmos or neo4j.")
             }
+            Error::DatabaseMismatch => {
+                write!(f, "The database connection you asked for does not match the configured database back-end.")
```

Comment:
It's a nit-picky stylistic thing, but I grew up with the idea that technical documentation should avoid second person.  (Yeah, I know... it appears in the doc comments, especially in the error documentation.)  Up to you... what's our style guide here?

Reply:
That didn't even occur to me. I have no objections to avoiding second person.

Reply:
Fixed.